bin/generate.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
We should probably add a note about this to the readme, so new contributors know how to run it.

Reply:
@BackEndTea it's already done in tests: https://github.com/webmozart/assert/pull/126/files#diff-6e2c75cbfedc47a231dafeeaa4c62fe7R161

Tests won't pass if the mixin does not match.

I don't mind having it in the readme as well though.

Reply:
This is more about making sure `bin/generate.php` is ignored by anyone cloning the repository: an inline comment suffices

Reply:
@zerkms as requested, clarification: what I mean is that a comment is needed at the top of this file.

something like:

```
/** this file is used by maintainers of the library to re-generate the type definitions of webmozart/assert: you probably don't need to use it */
```

Reply:
A comment like what @Ocramius suggested is good enough to indicate the file shouldn't be used by people using the package.

Reply:
@BackEndTea your original input was

> We should probably add a note about this to the readme, so new contributors know how to run it.

should we add `php bin/generate.php` command + its description to the readme, or not, given it's in unit tests already and this command is for developers only anyway.

Reply:
probably not needed